rules/test.bzl

```diff
@@ -124,6 +129,7 @@ def _ios_test(name, test_rule, test_suite_rule, apple_library, infoplists_by_bui
 
         rule(
             name = name,
+            module_name = module_name,
```

Comment:
I think this will break other people that have a different version of rules_apple ( similar of @mattrobmattrob's comment )